ibis/backends/tests/test_export.py

```diff
@@ -445,6 +445,10 @@ def test_arrow_timestamp_with_time_zone(alltypes):
 @pytest.mark.notimpl(
     ["impala"], raises=AttributeError, reason="missing `fetchmany` on the cursor"
 )
+@pytest.mark.xfail_version(
```

Comment:
Not sure if necessary, but just in case, cc'ing @deepyaman for visibility in case we use this in ibisml.